samples/Ark.ResourceWatcher/Ark.ResourceWatcher.Sample/Program.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2024 Ark Energy S.r.l. All rights reserved.
+// Licensed under the MIT License. See LICENSE file for license information.
+using Ark.ResourceWatcher.Sample.Config;
+using Ark.ResourceWatcher.Sample.Host;
+
+using Ark.Tools.NLog;
+using Ark.Tools.ResourceWatcher.WorkerHost.Hosting;
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Ark.ResourceWatcher.Sample;
+
+sealed class Program
```

Comment:
Run this program and ensure it runs without errors. This must be a valid scaffold starter